src/features/duplicates/hooks/useFieldSettings.ts

```diff
@@ -39,5 +39,13 @@ export default function useFieldSettings(duplicates: ZetkinPerson[]) {
     }
   });
 
-  return { fieldValues, initialOverrides };
+  function entryContainsOneValue(entry: Array<string | string[]>) {
+    return entry[1].length < 2;
+  }
```

Comment:
I think the name of this function is misleading, because it doesn't really check if there's just one value – it checks if there are less than two (which could also mean zero).